frontend/packages/dev-console/integration-tests/tests/git-import-flow.scenario.ts

```diff
@@ -8,18 +8,51 @@ import {
 import {
   navigateImportFromGit,
   enterGitRepoUrl,
-  addApplication,
   applicationName,
   appName,
   createButton,
+  setRouting,
+  setBuildConfig,
+  setDeployment,
+  setResources,
+  setLabel,
+  selectAdvancedOptions,
+  AdvancedOptions,
+  TLSTerminationValues,
+  buildConfigObj,
+  addNavigate,
+  routingObj,
+  deploymentObj,
+  resourceLimitsObj,
+  setScaling,
+  scalingObj,
+  setSecureRoute,
+  pipelineObj,
+  setPipelineForGitFlow,
+  importFromGitHeader,
+  addApplicationInGeneral,
+  gitUrlHelper,
+  builderImageSelected,
 } from '../views/git-import-flow.view';
 import { newApplicationName, newAppName } from '../views/new-app-name.view';
 import { switchPerspective, Perspective, sideHeader } from '../views/dev-perspective.view';
+import { pipelinecheckStatus, pipelineTableBody } from '../views/pipeline.view';
+import { elementByDataTestID, click, enterText } from '../utilities/elementInteractions';
+import { testData } from '../testData/git-import-flow.data';
+import { naviagteTo, NavigationMenu, ELEMENT_WAIT } from '../utilities/appFunctions';
+import {
+  verifyCreatedAppsInTopology,
+  selectActionInSideBar,
+  Actions,
+  topologyViewObj,
+  listViewObj,
+} from '../views/topology.view';
+
+const WAIT = ELEMENT_WAIT;
```

Comment:
Any reason for this rename? `const` variables can't be reassigned, and `ELEMENT_WAIT` is a number so it's fixed.

If you want to rename the variable for whatever reason to make it more contextually useful for this file, recommend you just rename it at the import level:

```
import { ..., ELEMENT_WAIT as WAIT } from '../utilities/appFunctions';
```

Would suggest you consider just not renaming it as it will make it easier to track and reason about when reading the code... but I'll leave that decision up to you.

Reply:
while import, renaming is a good idea.. Thanks a lot will use this comment :)